pkg/functions/runner.go

```diff
@@ -165,6 +167,106 @@ func runGo(ctx context.Context, job *Job) (err error) {
 	return
 }
 
+func runPython(ctx context.Context, job *Job) (err error) {
+	if job.verbose {
+		fmt.Printf("cd %v\n", job.Dir())
+	}
+
+	// TODO: should we allow the python binary location to be specified with
```

Comment:
how was that done before?

Reply:
These commands presume the binaries are available in `PATH`, and use the first match.  This is common.  However, a nice feature offered by some is to be able to override this on a case-by-case basis by setting environment variables such as `MYAPP_PYTHON=/path/to/python`.   We also support this in some cases, but not consistently throughout.  This might be a #good-first-issue